.github/workflows/ci-master-pr.yml

```diff
@@ -128,8 +128,11 @@ jobs:
     strategy:
       matrix:
```

Comment:
with #152, `games.json` may be the source of truth for the matrix, using [fromJSON](https://docs.github.com/en/actions/learn-github-actions/expressions#fromjson). this will work for now.

Reply:
Resolving